src/ui/QuoteMessageInput/QuoteMessageThumbnail.tsx

```diff
@@ -40,6 +59,18 @@ export default function QuoteMessageThumbnail({ message }: Props): ReactElement
         fixedSize
       />
     );
+  } else if (isVideoMessage(message as FileMessage) && thumbnailUrl) {
```

Comment:
```
if (isVideoMessage(message as FileMessage) && thumbnailUrl) {
   ...
} else if (isThumbnailMessage(message) || isMultipleFilesMessage(message)) {
   ...
}
```

I think we can make a bit simpler here by changing the order of these two.

Reply:
nice!